outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "cd_webhook" {
+value= "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
Note that this output value is sensitive - Terraform lets us [mark it with the sensitive flag](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) to ensure that it's treated as such and not logged to the console

Reply:
Updated, thank you